main/main.cpp

```diff
@@ -1958,6 +1958,14 @@ bool Main::start() {
 			ProjectSettings::get_singleton()->set_custom_property_info("rendering/canvas_textures/default_texture_repeat", PropertyInfo(Variant::INT, "rendering/canvas_textures/default_texture_repeat", PROPERTY_HINT_ENUM, "Disable,Enable,Mirror"));
 		}
 
+		if (editor) {
```

Comment:
Needs `#ifdef TOOLS_ENABLED`, this breaks non-tools build.

Reply:
Oops